run.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""This script provides command line access for testing forecasters on
+custom data sets.
+
+Example:
+    $python run.py --data-path "./data.csv" --test-fraction 0.1
```

Comment:
`run` is a poor name. Should use the package name, i.e. `deep4cast.py` for this.

Reply:
will make the changes.